sdk/core/core/test/cmocka/test_log.c

```diff
@@ -189,7 +189,7 @@ void test_az_log(void** state)
     // our code attempts to log a classification that's not in the customer's allow list.
     az_log_classification const classifications[] = { AZ_LOG_HTTP_REQUEST };
     az_log_set_classifications(
-        classifications, sizeof(classifications) / sizeof(classifications[0]));
+        classifications, _az_countof(classifications));
```

Comment:
If you want, you can ignore updating this one place. I have a PR in progress that replaces that with a single array without size, but with a terminating element.

Reply:
I did a find all replace all so no big deal. I can remove if it makes your merge easier but otherwise I can leave it.